apps/studio/components/interfaces/Auth/AuthTemplatesValidation.tsx

```diff
@@ -141,13 +124,8 @@ const EMAIL_CHANGE: FormSchema = {
     },
   },
   validationSchema: object().shape({
-    MAILER_SUBJECTS_EMAIL_CHANGE: string().required('"Subject heading is required.'),
+    MAILER_SUBJECTS_EMAIL_CHANGE: string().required('Subject heading is required.'),
   }),
-  misc: {
-    iconKey: 'email-icon2',
-    helper: `To complete setup, add this authorisation callback URL to your app's configuration in the Apple Developer Console.
-[Learn more](${DOCS_URL}/guides/auth/social-login/auth-apple#configure-your-services-id)`,
-  },
```

Comment:
@issuedat just checking that removing these `misc` properties of each item was purposeful?

Just double-checking as I have no idea what this is or what the implications are of removing it!